tallycash/extension
# Upgrade @ledgerhq

Closes #2554 

This PR Upgrade `@ledgerhq/errors` and `@ledgerhq/hw-app-eth`.  Probably solves the issue of setting up transactions for Ledger Nano S. 

**Issue:**
This issue happens sporadically. When setting up a transaction app is stuck on the below screen. Hitting refresh doesn't trigger any action.

<img width="240" alt="199907775-dc794ce6-6821-4551-97c0-40231679a7ae" src="https://user-images.githubusercontent.com/23117945/204764237-b13b33e1-f5c5-494a-9d21-bbc4e752b554.png">

## To Test

- [ ] check if the issue still occurs

        Steps to reproduce:
        
        1. Connect & Unlock Ledger
        2. Open the Ethereum App on Ledger
        3. Connect to a dApp with Tally Ho
        4. Set-up a transaction
- [ ] check that the update didn't mess anything up

Latest build: [extension-builds-2697](https://github.com/tallycash/extension/suites/9698136058/artifacts/464706927) (as of Tue, 06 Dec 2022 14:30:12 GMT).
